src/picongpu/include/particles/ionization/ionization.hpp

```diff
@@ -0,0 +1,397 @@
+/**
+ * Copyright 2014 Marco Garten, Axel Huebl, Heiko Burau, Rene Widera, 
+ *                  Richard Pausch, Felix Schmitt
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+/* includes from Particles.tpp */
+#include <iostream>
+
+#include "simulation_defines.hpp"
+#include "particles/Particles.hpp" 
+#include <cassert>
+
+
+#include "particles/Particles.kernel"
+
+#include "dataManagement/DataConnector.hpp"
+#include "mappings/kernel/AreaMapping.hpp"
+
+#include "particles/memory/buffers/ParticlesBuffer.hpp"
+#include "particles/ParticlesInit.kernel"
+#include "mappings/simulation/GridController.hpp"
+
+#include "simulationControl/MovingWindow.hpp"
+
+#include <assert.h>
+#include <limits>
+
+#include "traits/Resolve.hpp"
+
+//#include "particles/traits/GetIonizer.hpp"
+#include "types.h"
+
+#include "simulation_defines/param/speciesDefinition.param"
```

Comment:
Please remove this. This can have a lot side effects.
only include `#include "simulation_defines.hpp"`